exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/util.py

```diff
@@ -52,4 +40,4 @@ def _get_credentials(param):
     creds_env = environ.get(OTEL_EXPORTER_JAEGER_CERTIFICATE)
     if creds_env:
         return _load_credential_from_file(creds_env)
-    return ssl_channel_credentials()
+    return None
```

Comment:
How come we're making this change?

Reply:
I had asked this question here https://github.com/open-telemetry/opentelemetry-python/issues/1649#issuecomment-791177187 what should we in this case.

Users could do `JaegerExporter(..., credentials=ssl_channel_credentials(), ...)` to let grpc runtime pick the creds or  set the `OTEL_EXPORTER_JAEGER_CERTIFICATE` to certificate file path. If they don't do either of them then we use insecure channel.  This seemed the appropriate choice but willing to change if there are any suggestions.

Reply:
One way is to keep the `insecure` param and just remove the `OTEL_EXPORTER_*_ INSECURE`. WDYT? This also means we need to keep maintaining it and makes it difficult to remove later.

Reply:
Is this behavior defined anywhere? What are other languages doing?

Reply:
Looking at the go implementation I think we can keep the constructor param and just remove the env which is not part of spec.
```go
// WithInsecure disables client transport security for the exporter's gRPC connection
// just like grpc.WithInsecure() https://pkg.go.dev/google.golang.org/grpc#WithInsecure
// does. Note, by default, client security is required unless WithInsecure is used.
func WithInsecure() Option {
	return func(cfg *config) {
		cfg.canDialInsecure = true
	}
}
```
https://github.com/open-telemetry/opentelemetry-go/blob/8d8098146547e28913abfb3f1b228a99050c5563/exporters/otlp/otlpgrpc/options.go#L76-L83